config.py

```diff
@@ -39,11 +39,20 @@ class CIConfig(Config):
 
 
 class DevConfig(Config):
```

Comment:
As we're changing config in this PR can we also be consistent with the other services and use `DevelopmentConfig` here.